bundle/Resources/config/messenger.yaml

```diff
@@ -0,0 +1,52 @@
+# This is an example Symfony Messenger component configuration for asynchronous indexing.
+# Use it if it fits your needs, or create your own as you require.
+framework:
+    messenger:
+        reset_on_message: true
+        transports:
+            netgen_ibexa_asynchronous_indexing:
```

Comment:
`netgen_search_extra_asynchronous_indexing` ?

Reply:
I guess `netgen_ibexa_search_extra_asynchronous_indexing` if we want to be fully in sync with the naming. :)
I'll update.

Reply:
Updated in https://github.com/netgen/ibexa-search-extra/pull/4/commits/8fa9b22f442d0bcae9873cb91cca0df343a83754